Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -2433,3 +2433,62 @@
   - type: NpcFactionMember
     factions:
     - Passive
+
+- type: entity
+  name: kitten
+  parent: SimpleMobBase
```

Comment:
Is there any specific reason you parented it from SimpleMobBase instead of the existing MobCat prototype?

Reply:
uhm no, i change it later